docs/notebooks/annoytutorial.ipynb

```diff
@@ -145,22 +141,34 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "('a', 0.9973033787682652)\n",
-      "('in', 0.9971976953092963)\n",
-      "('an', 0.9971216244157404)\n",
-      "('which', 0.9964874279685318)\n",
-      "('has', 0.996418114984408)\n",
-      "CPU times: user 4 ms, sys: 0 ns, total: 4 ms\n",
-      "Wall time: 3.22 ms\n"
+      "Gensim: 0.007451029\n",
+      "Annoy: 0.002149934\n",
+      "\n",
+      "Annoy is 3.46570127269 times faster on average over 1000 random queries\n"
```

Comment:
The focus and emphasis on such a level of precision is misleading (and unnecessary).

Also, please mention the other factors that affect this number, like index size etc. So people don't go away thinking "annoy is ~3.5x faster than gensim", whereas in reality this is anything between 1x-infinity.